model/postgres/postgres_resource.rb

```diff
@@ -95,6 +95,14 @@ def set_firewall_rules
     private_subnet.firewalls.first.replace_firewall_rules(vm_firewall_rules)
   end
 
+  def ca_certificates
+    if root_cert_1.nil? || root_cert_2.nil?
```

Comment:
If one of them is not nil, don't we want to return one of them?

Reply:
I think in this case we want to return both the certs or none as the customer needs to trust both of them for things to run smoothly. Based on my understanding of the certificate init/refresh code, both of them are set together so this essentially just checks if the certs are initialized or not.